lib/matplotlib/axes/_base.py

```diff
@@ -1989,7 +1989,11 @@ def add_collection(self, collection, autolim=True):
             # Make sure viewLim is not stale (mostly to match
             # pre-lazy-autoscale behavior, which is not really better).
             self._unstale_viewLim()
-            self.update_datalim(collection.get_datalim(self.transData))
+            datalim = collection.get_datalim(self.transData)
```

Comment:
suggest a bit more comment here.  I had to look up what `minpos` was, and why you might want it here.

Reply:
Added.